### ISSUE TITLE
Erzya expert update is missing

### ISSUE THREAD

**Issue by SamPassmore** [kinbank/kinura#7]
The expert Erzya file is not in the dataset. 

Terhi has sent it to Sam in an excel sheet.

**Comment by tekahon** [comment]
The current Erzya data seems to be from IDS so as a result there are problems e.g. in how the relative age distinction is coded. Therefore I'd suggest replacing the current Erzya data with the list obtained from Nina where the relative age distinction is in the correct way.

**Comment by SamPassmore** [comment]
@tekahon could you provide me with a reference for Nina's transcription of the kinterms? 
So that I can replace the data.

**Comment by SamPassmore** [comment]
Hi terhi, 

There are two files Erzya and ErzyaMordvin. Should we be treating those as different?

**Comment by tekahon** [comment]
Erzya_erzy1239 (where erina_erzya_2015 is the source) is the old file, which should be replaced with Niina's sheet and put to the Varikin folder. 

With reference you mean what should be put to the sources.bib file? The expert is Niina Aasmäe who is a native speaker of Erzya and a linguist studying Erzya. She sent me the list in 2020. Let me know if you also need some other information for this.  

kinura_ErzyaMordvin_erzy1239 is the one collected by Niklas and should stay in Kinura repository (I assume this is the one you meant with ErzyaMordvin - at least I couldn't find a file with a name ErzyaMordvin without the kinura_ part in it).

**Comment by SamPassmore** [comment]
Does Niina have an academic affiliation? Or could we link them to a particular place where Ezrya is spoken? 

To be clear - we are treating Erzya and Erzya Mordovin as separate languages? (currently, they have the same glottocode).

**Comment by tekahon** [comment]
Yes, Niina is at the University of Tartu, at the Institute of Estonian and General linguistics.

Nope, they are one language (so it is correct that they have the same glottocode). There are two Mordvin languages, Erzya and Moksha, so sometimes people use just Erzya but sometimes Erzya Mordvin.
